API_request_test.py

```diff
@@ -123,13 +166,15 @@
     hours_meteostat.append(item['time'])
     temp_c_meteostat.append(item['temp'])
     rain_meteostat.append(item['prcp'])
-    
-print(hours_meteostat)
-print(temp_c_meteostat)
-print(rain_meteostat)
 
+# print(hours_meteostat)
+# print(temp_c_meteostat)
+# print(rain_meteostat)
+
+print(f'Temperature in {args.location} on {current_date}.')
 print('', temp_c_wheatherapi, '\n',temp_c_openmeteo, '\n', temp_c_meteostat)
 
+print(f'Rainfall in {args.location} on {current_date}.')
 print('', rain_wheatherapi, '\n', rain_openmeteo, '\n', rain_meteostat)
 
 print('', hours_wheatherapi[1], '\n', hours_openmeteo[1], '\n', hours_meteostat[1]) # three different time formats
```

Comment:
I like the fact you started using formatted strings:
```python
print(f'Temperature in {args.location} on {current_date}.')
```
..can we use it for all those?

Reply:
For all of those, no :P. The formatted string are used here as a header for printed statements from the lower line.

Reply:
Not sure if I understand, but ok, let's go back to it when you start the actual component implementation (I guess these will disappear then anyway).